recipes/llvm-bazel-cache/compile_bundle.py

```diff
@@ -0,0 +1,66 @@
+from pathlib import Path
+import ast
+import astor
+import os
+import shutil
+
+
+BAZEL_DIR = Path("utils") / "bazel"
+BAZEL_OUT = BAZEL_DIR / "bazel-bin" / "external" / "llvm-project"
+LIB_TARGET_DIR = Path(os.environ["PREFIX"]) / "lib"
+OTHER_TARGET_DIR = Path(os.environ["PREFIX"]) / "share" / "llvm_for_tf"
+
+
+def extend_linkopts(kw, name):
+    current_src = astor.to_source(kw.value)[:-1]
+    kw.value = ast.parse(f"{current_src} + ['-l{name}']").body[0].value
+
+
+def new_kwarg(name, code):
+    return ast.keyword(arg="linkopts", value=ast.parse(code).body[0].value)
+
+
+def rewrite_cc_library(symbol, node):
+    sym = symbol.lower()
```

Comment:
Can you write a short explanation here? AFAICT this modifies both the AST of `BUILD.bazel` _and_ copies libraries into `$PREFIX`. I don't under stand the role of the `llvm_for_tf` cache here, because the libraries get copied to `$PREFIX/lib` directly, which means that we might get conflicts/clobbering with other of our llvm-builds.

Also, I'd like to see in the logs which libs get copied into `$PREFIX` like that.

Reply:
I wrote some comments and added a logging statement.